src/components/GestureComponents.tsx

```diff
@@ -56,9 +94,13 @@ export type DrawerLayoutAndroid = typeof DrawerLayoutAndroid &
   RNDrawerLayoutAndroid;
 
 export const FlatList = React.forwardRef((props, ref) => {
+  const refreshControlGestureRef = React.useRef<any>();
```

Comment:
Why `any`?

Reply:
Changed the type and the value in https://github.com/software-mansion/react-native-gesture-handler/pull/2137/commits/5571228feea976af1751e93422f8e8d82f30b548